lib/cmd/fh3/credential/create.js

```diff
@@ -0,0 +1,105 @@
+    'certificateIos': i18n._("Param valid for IOS platform. Your iOS developer certificate file issued by Apple. You can download this file from your iOS developer portal. Path for the certificate file. (E.g --certificate:/Users/user/certificate)"),
+    'provisioning': i18n._("Param valid for IOS platform. A provisioning profile for a particular App ID downloaded from the iOS developer portal. The type of the provisioning profile should match the type selected above. A Distribution Certificate is used to build Release Apps for submission to the Android Marketplace. When you submit an App to the Android Marketplace, your certificate is used to identify you as the developer of the App Path for the provisioning file. (E.g --provisioning:/Users/user/provisioning)")
+  },
+  'url' : '/box/api/credentials',
+  'method' : 'post',
+  'customCmd' : function(params,cb) {
+    if (params.platform && params.platform !== 'ios' && params.platform !== 'android') {
+      return cb(i18n._('Invalid parameter for platform:') + params.platform);
+    }
+
+    var dataObject = [{
+      name: "bundleName",
+      value: params.bundleName
+    }, {
+      name: "platform",
+      value: params.platform
+    },{
+      name: "bundleType",
+      value: 'release'
+    }];
+
+    if (params.env) {
+      dataObject.push({
+        name: "environments",
+        value: params.env
+      });
+    }
+
+    if (params.platform === 'ios' || params.platform === 'android') {
+      if (!params.privateKey) {
+        return cb(i18n._('Missed parameter --privateKey.'));
+      }
+
+      dataObject.push({
+        name: "privatekey",
+        value: fs.createReadStream(params.privateKey),
+      });
+    }
+
+    if (params.platform === 'ios') {
+      if (params.bundleType && params.bundleType !== 'release' && params.bundleType !== 'distribution' &&   params.bundleType !== 'debug') {
+        return cb(i18n._('Invalid parameter for --bundleType :') + params.bundleType);
+      }
+
+      if (!params.provisioning) {
+        return cb(i18n._('Missed parameter --provisioning.'));
+      }
+
+      if (!params.certificateIos) {
+        return cb(i18n._('Missed parameter --certificateIos.'));
+      }
+
+      dataObject.push({
+        name: "certificate",
+        value: fs.createReadStream(params.certificateIos),
+      });
+
+      dataObject.push({
+        name: "provisioning",
+        value: fs.createReadStream(params.provisioning),
+      });
+
+      dataObject.push({
+        name: "provisioning",
+        value: fs.createReadStream(params.provisioning),
+      });
+
+    }
+
+    if (params.platform === 'android') {
+      if (!params.certificateAndroid) {
+        return cb(i18n._('Missed parameter --certificateAndroid.'));
+      }
+      dataObject.push({
+        name: "certificate",
+        value: fs.createReadStream(params.certificateAndroid),
+      });
+    }
+
+    fhreq.doRequestWithDataObjectValues(this.url,dataObject,cb);
```

Comment:
There is no message displayed to the user if the creation of credential bundle succeeded. If there is an error the user only see 'fhc Command executed with error'. It would be much more user friendly if there is a success message displayed, something like: 'Credential bundle created successfully.' In case of error proper error message should be displayed. Try to create a credential bundle with the name that already exists to reproduce this.
   
    fhreq.doRequestWithDataObjectValues(this.url,dataObject,'post',function(err, result) {
      if (err) {
        return cb(err);
      } else {
        console.log('Credential bundle created successfully.');
        return cb(null, result);
      }
    });

Note that for this to work you will have to change the doRequestWithDataObjectValues a little. Also using console.log is not a good practice, but it should ilustrate what needs to be done.